services/retentionmgr/metadataDep.go

```diff
@@ -192,15 +195,20 @@ func (t *metadataDepImpl) GetExtentInfo(destID destinationID, extID extentID) (e
 
 	extStats := resp.GetExtentStats()
 
-	extInfo = &extentInfo{
-		id:                extID,
-		status:            extStats.GetStatus(),
-		statusUpdatedTime: time.Unix(0, extStats.GetStatusUpdatedTimeMillis()*int64(time.Millisecond)),
-		storehosts:        make([]storehostID, 0, len(extStats.GetReplicaStats())),
-	}
+	storeUUIDs := make([]string, 0, len(extStats.GetReplicaStats()))
+	storehosts := make([]storehostID, 0, len(extStats.GetReplicaStats()))
 
 	for _, replicaStat := range extStats.GetReplicaStats() {
-		extInfo.storehosts = append(extInfo.storehosts, storehostID(replicaStat.GetStoreUUID()))
+		storehosts = append(storehosts, storehostID(replicaStat.GetStoreUUID()))
```

Comment:
what's the difference between the storehosts/storeUUIDs ? They both store the same set (except for the typecast)